server.js

```diff
@@ -1,32 +1,51 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
+import goldenGlobesData from './data/golden-globes.json'
 // import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+// Defining the port 
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+// Adding middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Starting defining routes 
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  /* res.json(goldenGlobesData) */
+  res.end('First backend project') 
 })
 
+// Filter based on title from code along with Van (netflix file)
+
+/* app.get('/shows', (req, res) => {
+  
+  const titleSearchString = req.query.title;
+
+  let filteredShows = netflixData;
+
+  if (titleSearchString) {
+    filteredShows = filteredShows.filter(item => {
+      const itemTitle = item.title.toString();
+        return itemTitle.includes(titleSearchString)
+    });
+  }
+
+  res.json(filteredShows);
+  }); */
```

Comment:
why is this commented out?

Reply:
Because I didn't want to work with netflix data, but  also didn't want to remove it, because it works and I can use it later for other projects.